validation/taylor_green_vortex_2d/validation_taylor_green_vortex_2d.jl

```diff
@@ -74,7 +81,7 @@ for density_calculator in density_calculators, perturbation in perturb_coordinat
     output_directory = joinpath("out_tgv", name_density_calculator * name_perturbation,
                                 wcsph ? "wcsph" : "edac",
                                 "validation_run_taylor_green_vortex_2d_nparticles_$(n_particles_xy)x$(n_particles_xy)")
-    saving_callback = SolutionSavingCallback(dt=0.02,
+    saving_callback = SolutionSavingCallback(dt=0.1,
```

Comment:
```suggestion
    saving_callback = SolutionSavingCallback(dt=1//10,
```
and also for the `PostprocessCallback`: `dt=2//100`.
This avoids approximation errors when to `affect` the periodic callback.